packages/bot-web-ui/src/components/dashboard/bot-builder/toolbox/toolbox.tsx

```diff
@@ -144,9 +146,11 @@ const Toolbox = ({
                                                                     }}
                                                                 >
                                                                     <Text size='xxs'>
-                                                                        {localize(
-                                                                            subCategory.getAttribute('name') as string
-                                                                        )}
+                                                                        {localize('{{name_attribute}}', {
+                                                                            name_attribute: subCategory.getAttribute(
+                                                                                'name'
+                                                                            ) as string,
```

Comment:
same for this

Reply:
yeah @niloofer-deriv we don't need it here, I removed it